requirements.txt

```diff
@@ -34,13 +34,15 @@ sentencepiece>=0.2.0
 # "old" version required for vLLM on CUDA to build
 tokenizers>=0.11.1
 toml>=0.10.2
-# Intel Gaudi 1.16.1 has torch 2.2.0a0+git8964477 with oneMKL
-torch>=2.3.0a0,<3.0.0 ; python_version == '3.10'
-torch>=2.3.0,<3.0.0 ; python_version != '3.10'
+# Intel Gaudi 1.16.2 has torch 2.2.0a0+git8964477 with oneMKL
```

Comment:
Does 1.16.2 still have `torch 2.2.0a0+git8964477`? I just want to make sure the comment is still accurate, also if not relevant we could remove it...

Reply:
I have changed the version to `2.2.2a0+gitxxx`. The git tag is not important, but the fact that it's a local pre-release is.